contrib/kubespray/script/k8s-generator.py

```diff
@@ -71,6 +71,13 @@ def generate_template_file(template_file_path, output_path, map_table):
     write_generated_file(output_path, generated_template)
 
 
+def path_handler(path):
+    if path[0] == '~':
```

Comment:
it could be `os.path.abspath(os.path.expanduser(path))` because of `os.path.expanduser` will return unchanged path if the path does not begin with a tilde

Reply:
Done